test 7/expectedOut.txt

```diff
@@ -0,0 +1,85 @@
+Activities
+(9003, 100, 101, 20200101)
+(9006, 60, 102, 20200101)
+(9003, -20, 1003, 20200201)
+(9006, -50, 1003, 20200201)
+(9003, -3, 1002, 20200201)
+(9003, -10, 1004, 20200301)
+(100293, 1000, 102, 20201201)
+(9003, -10, 1003, 20201201)
+(9006, -1, 1003, 20201202)
+(9006, -1, 1003, 20201202)
+(9006, -1, 1003, 20201202)
+(9006, -1, 1003, 20201202)
+(9006, -1, 1003, 20201202)
+(9006, -1, 1003, 20201202)
+(9006, -1, 1003, 20201202)
+(9006, -1, 1003, 20201202)
+(9006, -1, 1003, 20201202)
+(9006, -1, 1003, 20201202)
+(9003, -1, 1003, 20201202)
+(9003, -1, 1003, 20201202)
+(9003, -1, 1003, 20201202)
+(9003, -1, 1003, 20201202)
+(9003, -1, 1003, 20201202)
+(9003, -1, 1003, 20201202)
+(9003, -1, 1003, 20201202)
+(9003, -1, 1003, 20201202)
+(9003, -1, 1003, 20201202)
+(9003, -1, 1003, 20201202)
+Coffee stands
+(11, 'Bld-90', 1)
+(12, 'Bld-32', 2)
+(13, 'Bld-71', 1)
+Employees
+(1001, 'Alice', 45.5, 11)
+(1002, 'Bob', 45.5, 12)
+(1003, 'Charlie', 45.5, 12)
+(1004, 'Dan', 45.5, 13)
+Products
+(9001, 'Latte', 6.5, 0)
+(9002, 'Water', 7.0, 0)
+(9003, 'Magnum', 8.5, 47)
+(9004, 'Salad', 21.0, 0)
+(9005, 'Cafe & Mahafe', 11.5, 0)
+(9006, 'Chocolate', 8.5, 0)
+(100293, 'Ice Tea', 100.0, 1000)
+Suppliers
+(101, 'Strauss', '*6860 49 Hasivim St. Petach Tikva')
+(102, 'Osem', '1-700-70-76-76')
+
+Employees report
+Alice 45.5 Bld-90 0
+Bob 45.5 Bld-32 25.5
+Charlie 45.5 Bld-32 850.0
```

Comment:
that's all? maybe use bigger numbers? I want Charlie to make shit loads of money 🤑 😆